### PR TITLE
resend timer

### PR DESCRIPTION
**Pull request by avivbarniv**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/pages/Profile/Code.js [hunk 1]
```diff
@@ -55,14 +69,33 @@ const Code = () => {
...
+            <ResendButton
+              disabled={timer > 0}
+              onPress={() => {
+                if (timer === 0) {
+                  navigation.navigate('Phone');
+                }
+              }}
+            >
+              {i18n.t('onboarding.pages.code.resendCodeButton', {
+                seconds: timer,
```
Comment (OmerGery) [inline examples/client/Locomotion/src/pages/Profile/Code.js]: need the second param for the resendCodebutton? not only resendCodeTextSeconds?

File: examples/client/Locomotion/src/I18n/en.json [hunk 2]
```diff
@@ -57,14 +57,16 @@
                 "text": "Please enter code",
                 "subText": "A code has been sent to {{phoneNumber}}.",
                 "resendCodeText": "Didn't receive an SMS? ",
-                "resendCodeButton": "Resend"
+                "resendCodeButton": "Resend",
+                "resendCodeTextSeconds": " in {{seconds}} seconds"
```
Comment (OmerGery) [inline examples/client/Locomotion/src/I18n/en.json:61]: prefix spaces on purpose?

File: examples/client/Locomotion/src/pages/Profile/Code.js (hunk 1 above)
Comment (avivbarniv) [reply examples/client/Locomotion/src/pages/Profile/Code.js]: Right. fixed

File: examples/client/Locomotion/src/I18n/en.json (hunk 2 above)
Comment (avivbarniv) [reply examples/client/Locomotion/src/I18n/en.json:61]: Yes

**Pull request merged**